src/Microsoft.ML.Data/Transforms/FeatureContributionCalculationTransform.cs

```diff
@@ -25,10 +25,29 @@
 namespace Microsoft.ML.Data
 {
     /// <summary>
-    /// The FeatureContributionCalculationTransformer computes model-specific contribution scores for each feature.
+    /// The FeatureContributionCalculationTransformer computes model-specific per-feature contributions to the score of each data point.
     /// See the list of currently supported models below.
     /// </summary>
     /// <remarks>
+    /// <para>
+    /// Scorind a data set with a trained model produces a score, or prediction, for each data sample. To understand and explain these predictions
```

Comment:
>Scorind [](start = 8, length = 7)

typo: scoring

I suggest running this through a spell-checker (like copy-paste into Word) to catch the typos.